swap_meet/item.py

```diff
@@ -1,2 +1,30 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition = 0.0,):
+        self.category = category
+        self.condition = condition
+
+    # Doing this made all the tests fail due to missing arguments, but
+    # To add the age attribute, I think that I would:
+    # def __init__(self, age, category = "", condition = 0.0,):
+        # self.age = age
+        # self.category = category
+        # self.condition = condition
+
+    # and then for each item category, I would change to:
+    # def __init__(self, age, category="Electronics", condition=0.0):
+    #     super().__init__(age, category, condition)
+    
+    def condition_description(self):
+        if self.condition <= 1:
+            return "Terrible"
+        if 1 < self.condition <= 2: 
```

Comment:
Nice use of ranges to cover the possibility of non-integer conditions. Notice that we don't need the within range syntax here though. If we reached this line, it means that condition wasn't less than or equal to 1 (or it would have taken the first branch). In other words, we know that condition must at least be great than 1. So we don't need the lower part of the range and could write it simply as:
```python
        if self.condition <= 2: 
```

This is true for all of the ranges cascading down due to the order you wrote your checks!